### PR TITLE
fix: UI notifications details bugs

### PR DESCRIPTION
**Pull request by Jonathansoufer**
## **Description**

This PR fixes some UI bugs found on testing rounds of the new Notifications feature. 

- [x] Misaligned read/unread states
- [x] Missing network fee section
- [x] Reorg From/To address flows

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

### **After**

Screenshot 2024-09-02 at 18 54 03
Screenshot 2024-09-02 at 18 42 20
Screenshot 2024-09-02 at 17 07 19

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/util/notifications/notification-states/erc20-sent-received/erc20-sent-received.ts [hunk 1]
```diff
@@ -74,17 +78,13 @@ const state: NotificationState<ERC20Notification> = {
       fields: [
         {
           type: ModalFieldType.ADDRESS,
-          label: isSent(notification)
-            ? strings('notifications.modal.label_address_to_you')
-            : strings('notifications.modal.label_address_to'),
-          address: notification.data.to,
+          label: label_address_from(notification),
```
Comment (Prithpal-Sooriya) [inline app/util/notifications/notification-states/erc20-sent-received/erc20-sent-received.ts:81]: nice small cleanup!

File: app/util/notifications/notification-states/erc1155-sent-received/erc1155-sent-received.tsx [hunk 2]
```diff
@@ -86,17 +90,13 @@ const state: NotificationState<ERC1155Notification> = {
       fields: [
         {
           type: ModalFieldType.ADDRESS,
-          label: isSent(notification)
-            ? strings('notifications.modal.label_address_to_you')
-            : strings('notifications.modal.label_address_to'),
-          address: notification.data.to,
+          label: label_address_from(notification),
+          address: notification.data.from,
```
Comment (Prithpal-Sooriya) [inline app/util/notifications/notification-states/erc1155-sent-received/erc1155-sent-received.tsx:94]: Good fix. Yes the figma shows that it should be the "from" field first, then the "to" field.

File: app/util/notifications/notification-states/types/NotificationState.ts [hunk 3]
```diff
@@ -1,9 +1,52 @@
...
+const isSent = (
+  n:
+    | NativeSentReceiveNotification
+    | ERC20Notification
+    | ERC1155Notification
+    | ERC721Notification,
+) =>
+  n.type === TRIGGER_TYPES.ETH_SENT ||
+  n.type === TRIGGER_TYPES.ERC20_SENT ||
+  n.type === TRIGGER_TYPES.ERC721_SENT;
```
Comment (Prithpal-Sooriya) [inline app/util/notifications/notification-states/types/NotificationState.ts:42]: For completeness, also add `n.type === TRIGGER_TYPES.ERC1155_SENT`

**Review by Prithpal-Sooriya — DISMISSED** [review]
1 comment if you can resolve

File: app/util/notifications/notification-states/types/NotificationState.ts (hunk 3 above)
Comment (Jonathansoufer) [reply app/util/notifications/notification-states/types/NotificationState.ts:42]: fixed.

**Review by Prithpal-Sooriya — DISMISSED** [review]

File: app/components/Views/Notifications/Details/Fields/NetworkFeeField.tsx [hunk 4]
```diff
@@ -53,53 +62,89 @@ function NetworkFeeLabelAndValue(props: {
...
+            iconColor={IconColor.Info}
+          />
+
+          <View style={styles.boxLeft}>
+            <Text variant={TextVariant.BodyLGMedium}>
+              {strings('asset_details.network_fee')}
+            </Text>
+
+            <Text color={TextColor.Alternative} variant={TextVariant.BodyMD}>
+              {networkFee.transactionFeeInEth} ETH ($
```
Comment (tommasini) [inline app/components/Views/Notifications/Details/Fields/NetworkFeeField.tsx:95]: I can be missing something, but is this always in ETH? It shouldn't be the native token of the network?

Comment (Jonathansoufer) [reply app/components/Views/Notifications/Details/Fields/NetworkFeeField.tsx]: good catch. Support for multiple tickers added.

**Review by Prithpal-Sooriya — APPROVED** [review]

**Review by tommasini — APPROVED** [review]

**Pull request merged**